Add a.js client hints documentation (#4841)
segmentio/segment-docs: src/connections/sources/catalog/libraries/website/javascript/index.md

analytics.load('writekey', { disableAutoISOConversion: true })
```

> #### Client Hints

Reviewer: ```suggestion
#### Client hints
```

> Some `userAgent` strings are being frozen and will contain less information. If you would like to request more information when it's available, you can pass an array of strings with whichever fields you would like to request to the `highEntropyValuesClientHints` option (the example array contains all possible values).

Reviewer: ```suggestion
Some `userAgent` strings are frozen and contain less information. If you would like to request more information when it's available, you can pass an array of strings with fields you would like to request to the `highEntropyValuesClientHints` option. The example array below contains all possible values.
```
